src/Sylius/Bundle/ApiBundle/CommandHandler/Cart/PickupCartHandler.php

```diff
@@ -108,4 +110,15 @@ private function getLocaleCode(?string $localeCode, ChannelInterface $channel):
 
         return $localeCode;
     }
+
+    private function getDefaultAddress(CustomerInterface $customer): ?AddressInterface
+    {
+        $defaultAddress = $customer->getDefaultAddress();
+        if (null !== $defaultAddress) {
+            $clonedAddress = clone $defaultAddress;
+            $clonedAddress->setCustomer(null);
+        }
+
+        return $clonedAddress ?? null;
```

Comment:
We could reduce one "if" statement :P 
```suggestion
        $clonedAddress = null;
        
        if (null !== $defaultAddress) {
            $clonedAddress = clone $defaultAddress;
            $clonedAddress->setCustomer(null);
        }

        return $clonedAddress;
```
or

```suggestion
        if (null !== $defaultAddress) {
            $clonedAddress = clone $defaultAddress;
            $clonedAddress->setCustomer(null);
            
            return $clonedAddress;
        }

        return null;
```